### ISSUE TITLE
Incorrect quantized SSD mobiledet DSP model after quantization aware training

### ISSUE THREAD

**Issue by k8280627** [tensorflow/models#8726]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [x] I checked to make sure that this issue has not already been filed.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/research/object_detection

## 2. Describe the bug

I trained SSD+Mobiledet DSP model on a custom dataset, and did quantization aware training for 17K steps. However, when I exported the model to frozen graph using `export_tflite_ssd_graph.py`, the `FakeQuantWithMinMaxVars` that comes from `ReLU6` have min and max range larger than 0-6.  

https://github.com/tensorflow/models/issues/6112 this issue describes the similar issue, but it's not answered.

## 3. Steps to reproduce
1. train SSD+mobiledet DSP, with pretrained weights  stored in this link as fine-tune checkpoint:http://download.tensorflow.org/models/object_detection/ssdlite_mobiledet_dsp_320x320_coco_2020_05_19.tar.gz.
2. perform quantization aware training with the trained SSD+mobiledet DSP model as fine-tune checkpoint.
3. export the tflite graph using `export_tflite_ssd_graph.py`.

## 4. Expected behavior

The tflite graph would contain `FakeQuantWithMinMaxVars` that comes from `ReLU6` that have min and max range larger than 0-6.

## 6. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- Mobile device name if the issue happens on a mobile device:
- TensorFlow installed from (source or binary): pip install
- TensorFlow version (use command below): 1.15
- Python version: 3.7
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 10.0.130
- GPU model and memory: Nvdia GeForce RTX 2080

**Comment by k8280627** [comment]
Hi @tombstone @jch1 @pkulzc,

I noticed that the uint8 `mobile.tflite` in [this link](http://download.tensorflow.org/models/object_detection/ssdlite_mobiledet_dsp_320x320_coco_2020_05_19.tar.gz.) is properly quantized, as in the `FakeQuantWithMinMaxVars` that comes from `ReLU6` have the min and max range within 0-6. 

I was wondering, is this `model.tflite` quantized by quantization aware training (QAT) or post training quantization? If it's QAT, then could you let me know if there is any specific setting that I'd have to set or if there's anything that I might miss out? 

Another question is, for the quantization of the convolutional layer, is it per-tensor or per-axis quantization? Is there any option that I can force it to do per-tensor quantization, either with QAT of post training quantization?

Thank you very much for your help!! Really appreciate it very much!

**Comment by k8280627** [comment]
Hi @tombstone @jch1 @pkulzc,

just wanted to bump this up and see if there's any update on this issue. Please let me know if you need additional information.
